modules/livewrappedBidAdapter.js

```diff
@@ -177,14 +184,33 @@ function hasPubcid(bid) {
 }
 
 function bidToAdRequest(bid) {
-  return {
+  var adRequest = {
     adUnitId: bid.params.adUnitId,
     callerAdUnitId: bid.params.adUnitName || bid.adUnitCode || bid.placementCode,
     bidId: bid.bidId,
     transactionId: bid.transactionId,
-    formats: bid.sizes.map(sizeToFormat),
+    formats: getSizes(bid).map(sizeToFormat),
     options: bid.params.options
   };
+
+  if (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.native) {
+    adRequest.banner = true;
```

Comment:
Why does setting this matter if `native` is in the mediaTypes?

Reply:
It's related to how it works in the backend. It only needs setting if both banner and native is enabled, otherwise (if not native) banner is implied.

Btw. thanks for all the input, I will fix those things for the next PR.